http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/ProxyService.scala

```diff
@@ -194,16 +227,21 @@ class ProxyService(
   ): IO[AppSamResourceId] =
     for {
       ctx <- ev.ask[AppContext]
-      cacheResult <- samResourceCache.cachingF(key)(None)(
+      cacheSamResourceId <- samResourceCache.cachingF(key)(None)(
         getSamResourceFromDb(key)
       )
-      resourceId = cacheResult.map(AppSamResourceId)
+      // TODO modify the cache to store more than just the App Resource ID string
+      // TODO REENABLE CACHE IMMEDIATELY AFTER FIGURING OUT THE ABOVE
+      cacheAppAccessScopeString <- getSamAppAccessScopeFromDb(key)
```

Comment:
This is an issue I ran into because of how the cache is being mocked for unit tests. The cache can only contain one string the resource id, but we also need to retrieve the access scope as well now. Feels like this opens the door to unit tests doing some IO stuff, which we should avoid. i would love some help on refactoring the cache :)

Reply:
As discussed earlier, we can probably update the cache value to contain multiple fields as `(String, String)` or something.

Reply:
Done, the cache now contains a tuple with both the string id and the appAccessScope :)